src/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx

```diff
@@ -37,32 +34,23 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
   }
 
   const savedSearchEntity: SavedSearchEntity = {
-    placeholder: artist.name ?? "",
+    placeholder: artist.name!,
     owner: {
       type: OwnerType.artist,
       id: artist.internalID,
-      name: artist.name ?? "",
+      name: artist.name!,
       slug: artist.slug,
     },
     defaultCriteria: {
       artistIDs: [
         {
-          displayValue: artist.name ?? "",
+          displayValue: artist.name!,
           value: artist.internalID,
         },
       ],
     },
   }
 
-  const defaultPills: FilterPill[] = [
-    {
-      isDefault: true,
-      value: artist.internalID,
-      displayValue: artist.name ?? "",
-      field: "artistIDs",
-    },
-  ]
```

Comment:
We're removing this notion of a 'default pill' from the filter section; though it remains to be displayed in the alert dialog.

The biggest puzzle to me is why we were manually providing this as a prop, when the dialog just pulls it out of the filter state. 🤷